docs/04-operation-guides/troubleshooting/eventing/evnt-05-fix-pending-messages.md

```diff
@@ -8,139 +8,138 @@ You publish events, but some of them are not received by the subscriber and stay
 
 ## Cause
 
-When NATS EventingBackend has more than 1 replica and the `Clustering` property on the NATS Server is enabled,
-a leader-election is taking place on the stream and consumer levels (
-see [NATS Documentation](https://docs.nats.io/running-a-nats-service/configuration/clustering/jetstream_clustering)).
-Once the leader is elected, all the messages are being replicated across the replicas.
+When the NATS EventingBackend has more than 1 replica, and the `Clustering` property on the NATS Server is enabled, one replica is elected as leader on the stream and consumer levels (see [NATS Documentation](https://docs.nats.io/running-a-nats-service/configuration/clustering/jetstream_clustering)).
+When the leader is elected, all the messages are replicated across the replicas.
 
-Sometimes replicas can go out-of-sync with the other replicas.
+Sometimes replicas can go out of sync with the other replicas.
 As a result of this, messages on some consumers can stop being acknowledged and start piling up in the stream.
```

Comment:
I know you did not change that in that PR, but maybe we could make the sentence a little bit shorter as well?
```suggestion
As a result, messages on some consumers can stop being acknowledged and start piling up in the stream.
```

Reply:
Very nice!